fe/fe-core/src/main/java/org/apache/doris/planner/external/IcebergScanProvider.java

```diff
@@ -136,16 +177,33 @@ public List<InputSplit> getSplits(List<Expr> exprs) throws UserException {
         List<InputSplit> splits = new ArrayList<>();
 
         int formatVersion = ((BaseTable) table).operations().current().formatVersion();
+        DeleteFileTempTable deleteFileTable = null;
+        if (formatVersion >= MIN_DELETE_FILE_SUPPORT_VERSION) {
+            String dbName = ClusterNamespace.getFullName(analyzer.getClusterName(), hmsTable.getDbName());
+            deleteFileTable = new DeleteFileTempTable(hmsTable.getCatalog(),
```

Comment:
You don't need to create `DeleteFileTempTable` for every split.
You can just create a final static member of this table somewhere, Because it just save a schema of delete file.